libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/SwipeToRefreshHelper.java

```diff
@@ -18,19 +17,58 @@
         public void onRefreshStarted();
     }
 
+    /**
+     * Helps {@link org.wordpress.android.util.widgets.CustomSwipeRefreshLayout} by passing the
+     * {@link android.support.v4.widget.SwipeRefreshLayout}, {@link RefreshListener}, and color.
+     *
+     * @param context               {@link Context} in which this layout is used.
+     * @param swipeRefreshLayout    {@link CustomSwipeRefreshLayout} for refreshing the contents
+     *                              of a view via a vertical swipe gesture.
+     * @param listener              {@link RefreshListener} notified when a refresh is triggered
+     *                              via the swipe gesture.
+     *
+     * @deprecated Use {@link #SwipeToRefreshHelper(CustomSwipeRefreshLayout, RefreshListener, int...)} instead.
+     */
+    @Deprecated
     public SwipeToRefreshHelper(Context context, CustomSwipeRefreshLayout swipeRefreshLayout, RefreshListener listener) {
-        init(context, swipeRefreshLayout, listener);
+        init(swipeRefreshLayout, listener, android.R.color.holo_blue_dark);
```

Comment:
This constructor `SwipeToRefreshHelper(Context context, ...)` now seems to never be used. Perhaps we can remove it?

Reply:
It's not used in the [WordPress-Android](https://github.com/wordpress-mobile/WordPress-Android/) app, but it could be used in other apps.  That's why I added the `@Deprecated` annotation and the `@deprecated Use {@link #SwipeToRefreshHelper(CustomSwipeRefreshLayout, RefreshListener, int...)} instead.` comment.  That's a safer strategy to support backward compatibility rather than removing methods and breaking builds when apps update the library dependency.  We can remove it next time we release a major version of the [WordPress-Utils-Android](https://github.com/wordpress-mobile/WordPress-Utils-Android/) library.